netty-utils/src/test/java/org/asynchttpclient/netty/util/ByteBufUtilsTests.java

```diff
@@ -0,0 +1,71 @@
+package org.asynchttpclient.netty.util;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import java.nio.charset.Charset;
+import org.testng.annotations.Test;
+import org.testng.Assert;
+import org.testng.internal.junit.ArrayAsserts;
+
+public class ByteBufUtilsTests {
+
+    @Test
+    public void testByteBuf2Bytes() {
+        ByteBuf byteBuf = Unpooled.copiedBuffer(new byte[]{'f', 'o', 'o'});
```

Comment:
Use wrappedBuffer instead of copiedBuffer

Reply:
All of the ByteBuf instances you create must be released, otherwise Netty will complain about memory leaks.

Reply:
Instances of `copiedBuffer` has been replaced with `wrappedBuffer`
`ByteBuf.release()` called before the end of the method